_posts/2019-12-31-memkind-dax-kmem.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Memkind support for KMEM DAX option
+author: michalbiesek
+layout: post
+identifier: memkind-support-for-KMEM-DAX-option
+---
+
+### Introduction
+
+Linux kernel version 5.1 brings in [support][kernel-patch] for the volatile-use of persistent memory
+as a hotplugged memory region (**KMEM DAX**).
+From the perspective of OS, persistent memory will be seen as a separate NUMA node(s).
+[libmemkind][memkind-release] extends its API by new kinds which allow for automatic detecting and allocating to
```

Comment:
automatic detection and allocation from these new Persistent Memory NUMA nodes.

Reply:
Done